dev-docs/conventions.md

```diff
@@ -35,43 +35,41 @@ Further we try to adhere to the following guidelines:
   Example:
 
   ```Go
-  log.Infof("This is a log message")
-  log.Infof("Waiting to do something...")
+  log.Info("This is a log message")
+  log.Info("Waiting to do something...")
   log.Error("A critical error occurred!")
   ```
 
-* Use the `With()` method to add structured context to your log messages. The context tags should be easily searchable to allow for easy log filtering. Try to keep consistent tag naming!
+* Use the `WithAttrs()` method to add structured context to your log messages. The context tags should be easily searchable to allow for easy log filtering. Try to keep consistent tag naming!
 
   Example:
 
   ```Go
-  log.With(zap.Error(someError), zap.String("ip", "192.0.2.1")).Errorf("Connecting to IP failed")
+  log.WithAttrs(slog.Any("error" someError), slog.String("ip", "192.0.2.1")).Error("Connecting to IP failed")
   ```
 
-* Log messages may use format strings to produce human readable messages. However, the information should also be present as structured context fields if it might be valuable for debugging purposes.
+* Slog does not support format strings out of the box. You have to wrap any format strings in `fmt.Sprintf()`.
```

Comment:
IMO, with slog, we should rather advise to not use format strings, as these are not machine-readable in case of switching to JSON loggers. Instead, where a format string was used previously, the input should now be supplied as an attribute. E.g.:
```go
log.withAttrs(slog.String("address", addr), slog.Int("port", port)).Info("Starting server")
```